Java/core/src/main/java/com/amazon/randomcutforest/executor/AbstractForestUpdateExecutor.java

```diff
@@ -58,14 +58,15 @@ protected AbstractForestUpdateExecutor(IUpdateCoordinator<PointReference, Point>
      * @param point The point used to update the forest.
      */
     public void update(double[] point) {
-        update(point, updateCoordinator.getTotalUpdates());
+        update(point, stateCoordinator.getTotalUpdates());
     }
 
     public void update(double[] point, long sequenceNumber) {
-        double[] pointCopy = cleanCopy(point);
-        PointReference updateInput = updateCoordinator.initUpdate(pointCopy, sequenceNumber);
-        List<UpdateResult<PointReference>> results = update(updateInput, sequenceNumber);
-        updateCoordinator.completeUpdate(results, updateInput);
+        double[] pointCopy = RandomCutForest.cleanCopy(point);
```

Comment:
Given that the `cleanCopy` method is in the forest, why don't we call it in the forest method before passing the point here?

Reply:
Done.